### DISCUSSION TITLE
PrusaSlicer Profile Bundle

### DISCUSSION THREAD

**Discussion by sadtitled** [bassamanator/Sovol-SV06-firmware#45] [Ideas]
Hey @bassamanator ,

Was wondering if it's possible to create a prusa slicer config ini from the super slicer settings. Is it as easy as just opening it up and copying the settings over? Do the profiles work automatically?

**Comment by bassamanator** [comment]
You can import a SS config bundle into PS. Whether this will work properly is unknown.

**Comment by armishmajerj** [comment]
When I've tried importing the provided SS configs into PR I couldn't get it to work. Granted I spent 0 time at all trying to figure it out :) I can just comment that it doesn't immediately work. I believe there are different configs/flags between the two that is causing the error. I wish PS would allow you to still import what it can figure out and highlight what it had no idea what to do with.

**Comment by tbonge** [comment]
Some of the SS settings don't use the same range of values and it will be way off in PS. I translated over the settings for PS. You can import them both and compare then side by side to see the difference. The only major change I made is a thicker first layer. Also, you need to change the pressure advance custom gcode to your own calibrated values.

[SV06 PrusaSlicer_config_bundle.zip](https://github.com/bassamanator/Sovol-SV06-firmware/files/11305425/SV06.PrusaSlicer_config_bundle.zip)

**Reply by armishmajerj** [reply]
Oooh I'll have to give that a try! I've wanted to play around more with organic supports in the PS alpha. The one print I did came out just fine but there were definitely issues with the print because I hastily did the settings. Thanks!

**Reply by Trilis29** [reply]
can you migrate 45degree profile aswell?

**Reply by ChrSch84** [reply]
+1

**Comment by SV06PLUSNOOB** [comment]
I have imported the superslicer bundle in Prusaslicer. Only error i get is: external perimeter extrusion width=0.2 mm is too low to be printable at a layer height 0.2 mm. Is 115% an ok value ?

**Reply by tbonge** [reply]
Try using the prusaslicer bundle I made, linked above. I took the settings from the superslicer bundle and use them to modify the i3 MK3S preset in prusaslicer.

**Reply by Jayksopp** [reply]
prusa slicer sets width based on layer height iso nozzle width. You have to change the width settings to something appropriate. I set them all to 0 myself and let prusa slicer use the default values.
![image](https://github.com/bassamanator/Sovol-SV06-firmware/assets/96485220/a621decb-e6c4-4a8d-98f5-7f2839cab143)

**Reply by Trilis29** [reply]
just do simple math 0.4 * xx% and you will haev correct answer in Prusa :)

**Comment by Trilis29** [comment]
I see prusaslicer config retract speed is 35 instead of 50?

**Reply by bassamanator** [reply]
The appropriate value for this extruder is 45-55mm/s for retraction. Some more brittle filaments (e.g., matte) will require a slower retraction. Feel free to adjust and override per filament.

**Comment by jaum20** [comment]
I am using @tbonge configs for PrusaSlicer and it is working fine (thanks for sharing), but there are some minor problems. Thumbs are not showing in Mainsail (but are showing in the [moonraker telegram app](https://github.com/nlef/moonraker-telegram-bot)). Also, mainsail is showing some gcode related errors: `Unknown command:"M900"`. No sure if those are related to slicer settings tho

**Reply by shadowframe** [reply]
I am using now the presets at this repro for ps. But if you want to adjust it : https://docs.mainsail.xyz/overview/slicer/prusaslicer

**Comment by shadowframe** [comment]
I am searching for a script prusaslicer to orcaslicer preset transfer. If I can’t find a script I will manual transfer the attributes. Must got more time 😂

**Reply by bassamanator** [reply]
Since OS is based on PS, you should be able to import a PS or SS config bundle into OS.

**Comment by jaum20** [comment]
I am using this profile and it is great, but I don't like the printer doing a mash calibrate before every single print. How can I disable this?

**Reply by bassamanator** [reply]
What profile exactly?

**Reply by jaum20** [reply]
Sorry. It's the KAMP settings that makes the printer do a mesh calibrate every time. It is intended to do a full mesh calibration before every print irrespective to the object size?

**Reply by bassamanator** [reply]
If you have KAMP enabled, it's going to do a mesh calibrate before every print based on the size of the print.

The config bundles and OSS config are separate entities. I recommend you read FAQs.

**Comment by jaum20** [comment]
> it's going to do a mesh calibrate before every print based on the size of the print.

So it shouldn't do a 'full' mesh? Because mine do a full mesh calibrate every time irrespective to the size of the print
